infra-as-code/bicep/modules/policy/assignments/alzDefaults/mc-alzDefaultPolicyAssignments.bicep

```diff
@@ -0,0 +1,732 @@
+@description('Prefix for the management group hierarchy. DEFAULT VALUE = alz')
+@minLength(2)
+@maxLength(10)
+param parTopLevelManagementGroupPrefix string = 'alz'
+
+@description('The region where the Log Analytics Workspace & Automation Account are deployed. DEFAULT VALUE = chinaeast2')
+param parLogAnalyticsWorkSpaceAndAutomationAccountLocation string = 'chinaeast2'
+
+@description('Log Analytics Workspace Resource ID. - DEFAULT VALUE: Empty String ')
+param parLogAnalyticsWorkspaceResourceID string = ''
+
+@description('Number of days of log retention for Log Analytics Workspace. - DEFAULT VALUE: 365')
+param parLogAnalyticsWorkspaceLogRetentionInDays string = '365'
+
+@description('Automation account name. - DEFAULT VALUE: alz-automation-account')
+param parAutomationAccountName string = 'alz-automation-account'
+
+@description('An e-mail address that you want Microsoft Defender for Cloud alerts to be sent to.')
+param parMsDefenderForCloudEmailSecurityContact string = 'security_contact@replace_me.com'
+
+@description('ID of the DdosProtectionPlan which will be applied to the Virtual Networks. If left empty, the policy Enable-DDoS-VNET will not be assigned at connectivity or landing zone Management Groups to avoid VNET deployment issues. Default: Empty String')
```

Comment:
Considering that ddos is not supported in mc we should remove this param

Reply:
We'll keep this as a placeholder until DDoS Protection becomes available in China North 3 and China East 3 regions.
<img width="647" alt="image" src="https://user-images.githubusercontent.com/8053228/176101466-3c67c0d2-24e6-4933-8a01-477a8a9e375f.png">